src/calculateRentalCost.js

```diff
@@ -28,7 +28,25 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayCost = 40;
+  const threeDayBonus = 20;
+  const sevenDayBonus = 50;
+  const costWithoutBonus = days * dayCost;
+  let totalCost;
+
+  if (days < 3) {
+    totalCost = costWithoutBonus;
+  }
```

Comment:
Useless condition

Reply:
fixed + assigned total as default with ability to be changed